applications/high_speed_endoscopy/README.md

```diff
@@ -29,6 +29,39 @@ recommended hardware, GSYNC and RDMA enabled on exclusive display mode is 10ms o
 Clara AGX Devkit and 8ms on NVIDIA IGX Orin DevKit ES. This is the photon-to-glass latency of
 a frame from scene acquisition to display on monitor.
 
+## Building and Running the Application
+
+This application cannot be compiled within the Holohub container. It needs to be compiled on the 
+host machine.
+
+### Set up the environment
+
+```bash
+export LD_LIBRARY_PATH=/usr/lib/aarch64-linux-gnu:$LD_LIBRARY_PATH
```

Comment:
Typically handled by `/etc/ld.so.conf.d/aarch64-linux-gnu.conf` (at least it is on my IGX), can you confirm this is needed?

Can check:
```
ldconfig -p | grep /usr/lib/aarch64-linux-gnu   # should see at least one library listed
```

Reply:
If this is required on your IGX, it suggest the IGX might have an issue with its setup and needs reflash..

Reply:
Thanks @sohamm17  for confirming!